app/packages/core/src/components/Modal/Modal.tsx

```diff
@@ -199,9 +199,9 @@ const SampleModal = () => {
               {jsonPanel.isOpen && (
                 <JSONPanel
                   containerRef={jsonPanel.containerRef}
-                  jsonHTML={jsonPanel.jsonHTML}
```

Comment:
Is `jsonHTML` no longer used? Maybe `useJSONPanel` needs some cleanup?

Reply:
agreed!